spec/expression.dd

```diff
@@ -2499,6 +2512,13 @@ $(H3 $(LNAME2 uniform_construction_syntax, Uniform construction syntax for built
         auto b = wchar();   // same as: wchar.init
         ---
 
+    $(P The argument may not be given a name:)
+
+        ---
+        version (none) auto a = short(x: 1); // Allowed, parses according to grammar
```

Comment:
Is this `version(none)` line intended? Seems to be identical to the following line, except for the comment.

Reply:
It's meant to show that adding a named argument is a semantic error, not a parser error. Code inside `version (none)` is parsed but not semantically analyzed, so it is allowed there. I wonder if I could make that clearer, or if I should just remove that since it's an unimportant detail to draw attention to.

Reply:
I think this is an implementation detail that's not relevant to the user.